designs/administrator/future-openshift/clickable-badge/index.md

```diff
@@ -0,0 +1,43 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Clickable Badges
+
+Resource badges can be made clickable to expose further info about the resources.
+By making the badge clickable, we can surface further information pulled from the "Explore" menu item, thus surfacing the "Explore" content on the go.
+This could also reduce confusion around acronyms.
+
+
+
+## General behaviour
+![Clicking on a badge](img/Clickable-Badge-00.png)
+* Currently, resources are implemented to have the name clickable. Badges are currently not clickable and detached from the resource link.    
+* On clicking the resource badge, a popover appears displaying the following:
+  * Full acronym breakdown as the title of the popover.
+  * Resource description, pulled from the Explore>Resource>Details>Description.
+  * Link to the Explore>Resource page.
+
+### Specs
+
+![img](img/Clickable-Badge-02.png)
+* All badges placed in-page or within a modal can be made clickable, including the following placements:
+  * Resource details page title
+  * Within a list.
+  * Within an infobox
+  * Within a modal (in page)
+* Badges that appear on drop-downs should be excluded and not be made clickable.
+* the badge popover should be triggered by clicking, but also by hovering with a 0.5 seconds timeout.
+
+![img](img/Clickable-Badge-01.png)
+* The badge should have a clickable active area with the same height as a primary button
+* This active area should be the same length as the badge (no margins).
+* On hover, the curser should be changed to "clickable object"
+* In the current badge implementation (OS 4.8), no visual change will be made to the badge on hover or click.
+
+
+## Future expansions
+![img](img/Clickable-Badge-03.png)
```

Comment:
What type of panel is this? 
How will this look when a Quick Start is in progress?
What happens if in the topology view, the user selects an object in topology & hits the resource badge on the side panel?
It feels like we could now have (1) nav (2) content area (3) side panel (4) quick start panel (5) badge panel ?

Reply:
Sorry for the late response.
I've updated the PR to address the issues you've raised.
In general:
On the Admin side, the badge is decoupled from the name. Making the badge clickable should coexist with clicking on the workload name. This means there are two side panels:
1. Workload side panel
2. 'Explore badge' side panel
Only one of them should be open, so opening one will close the other.
Because of the way the workload on the topology view looks like, the badge will be coupled with the name. So the Badge side panel is not available at the topology view.
However, it should be available on the list view of that topology. Today, the Dev and Admin console are not aligned with what part of the workload is clickable on the list view. This should be a good opportunity to fix that.
Like the workload side panel, the Badge side panel should coexist with Quickstart. Both should be open at the same time.